core/templates/pages/about-page/about-page.component.spec.ts

```diff
@@ -0,0 +1,180 @@
+// // Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Why is this file commented out?

Reply:
I mostly wanted to get the design approved before getting into the tests. I will change all of these now.

Reply:
Changed the spec!